### PR TITLE
feat: added quiz functionality

### PR DESCRIPTION
**Pull request by uday-jangra**

### CODE REVIEW

File: apps/server/src/routes/quiz.router.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    console.log(err);
+    res.status(500).json({ message: err.message })
+  }
+})
+
+quizRouter.post('/submit', async (req: IRequest, res: Response) => {
+  try {
+    console.log(req.body)
+    const { userQuizId } = req.body
+    const quizAnswers = req.body.quizAnswers
```
Comment (regmimilan) [inline apps/server/src/routes/quiz.router.ts:22]: Add or check if user quiz id or answer is null

File: apps/web/src/components/QuizForm.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+const QuizForm = ({
+  quiz,
+  quizAnswers,
+  setQuizAnswers,
+  submitQuiz,
+}: QuizFormProps) => {
+  const [questionIndex, setQuestionIndex] = useState(0)
+  const [answer, setAnswer] = useState('')
+  console.log(quizAnswers)
```
Comment (regmimilan) [inline apps/web/src/components/QuizForm.tsx:22]: Remove console.log

File: apps/server/src/routes/quiz.router.ts (hunk 1 above)
Comment (uday-jangra) [reply apps/server/src/routes/quiz.router.ts:22]: resolved

File: apps/web/src/components/QuizForm.tsx (hunk 2 above)
Comment (uday-jangra) [reply apps/web/src/components/QuizForm.tsx:22]: resolved.

**Review by regmimilan — APPROVED** [review]

**Pull request merged**